Modules/_csv.c

```diff
@@ -1442,10 +1459,22 @@ csv_register_dialect(PyObject *module, PyObject *args, PyObject *kwargs)
     Py_RETURN_NONE;
 }
 
+
+/*[clinic input]
+_csv.unregister_dialect
+
+    name: object
```

Comment:
I suppose this doesn't say `str` because we need the `PyObject` inside the function. Any way we can still get the `__text_signature__` more precise?

Reply:
Not that I'm aware of (within Argument Clinic and without changing some csv.Errors to TypeErrors)